theano/gof/op.py

```diff
@@ -714,7 +738,7 @@ def R_op(self, inputs, eval_points):
             "own op, implement the R_op method." %
             (self, self.__class__.__name__))
 
-    def perform(self, node, inputs, output_storage):
+    def perform(self, node, inputs, output_storage, params=None):
```

Comment:
It is probably a left over from rebase? Anyway, it is like this in the current master. If you rebase, it should go away.